boards/arm/twr_ke18f/twr_ke18f.dts

```diff
@@ -122,6 +122,20 @@
 	};
 };
 
+&cpu0 {
+	clock-frequency = <120000000>;
+	cpu-power-states = <&idle &stop>;
+};
+
+&idle {
```

Comment:
shouldn't this be enclosed in power-states { }?

Reply:
No. The power-states are defined on the SoC level DTSI, but the time it takes to enter/leave the given state is board (clock setup) dependent, so those are defined at the board level (with reference to the power-states defined at the SoC level).